codec-http2/src/main/java/io/netty/handler/codec/http2/WeightedFairQueueByteDistributor.java

```diff
@@ -256,19 +255,36 @@ void isActiveCountChangeForTree(int increment) {
             activeCountForTree += increment;
             if (!stream.isRoot()) {
                 State pState = state(stream.parent());
+                assert activeCountForTree != increment ||
```

Comment:
FYI, the name of this method (`isActiveCountChangeForTree`) is a bit confusing ... it sounds like it should be a `boolean`.

Maybe we should fix that in a follow-on PR?